Src/DependencyCollector/Shared/Implementation/DesktopDiagnosticSourceHttpProcessing.cs

```diff
@@ -0,0 +1,81 @@
+﻿#if NET45
+namespace Microsoft.ApplicationInsights.DependencyCollector.Implementation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Net;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.ApplicationInsights.DependencyCollector.Implementation.Operation;
+    using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.ApplicationInsights.Web.Implementation;
+
+    /// <summary>
+    /// Concrete class with all processing logic to generate RDD data from the callbacks received from HttpDesktopDiagnosticSourceListener.
+    /// </summary>
+    internal sealed class DesktopDiagnosticSourceHttpProcessing : HttpProcessing
+    {
+        internal ObjectInstanceBasedOperationHolder TelemetryTable;
+
+        internal DesktopDiagnosticSourceHttpProcessing(TelemetryConfiguration configuration, ObjectInstanceBasedOperationHolder telemetryTupleHolder, bool setCorrelationHeaders, ICollection<string> correlationDomainExclusionList, string appIdEndpoint)
+            : base(configuration, SdkVersionUtils.GetSdkVersion("rdd" + RddSource.FrameworkAndDiagnostic + ":"), null, setCorrelationHeaders, correlationDomainExclusionList, appIdEndpoint)
```

Comment:
Please check with Sergey if "rdd" should be some other prefix code for uniqueness or not.